tfx_addons/xgboost_evaluator/xgboost_predict_extractor.py

```diff
@@ -22,9 +22,16 @@
 import numpy as np
```

Comment:
@casassg Why do we need these changes (`xgboost_evaluator` and `xgboost_evaluator_test`)?

Reply:
landed upstream fixes here. This reduces the PR to only feast changes